sections/method/topic_modelling.tex

```diff
@@ -3,7 +3,7 @@ \subsection{Topic Modelling}\label{subsec:topicModelling}
 The result consists of a topic-word distribution matrix $\beta$, which for each topic gives a distribution of words belonging to said topic, and a document-topic distribution matrix $\theta$ which for each document gives a distribution of topics to which the document belongs to.
 
 \subsubsection{\acrlong{lda}}\label{subsec:lda}
-\todo{cite lda} introduce \gls{lda}, which has since become a staple within topic modeling.
+\cite{lda} introduce \gls{lda}, which has since become a staple within topic modeling.
```

Comment:
```suggestion
\citeauthor{lda} \cite{lda} introduce \gls{lda}, which has since become a staple within topic modeling.
```
Så bliver det noget lignende "Blei et al. [1] introduce ..."